test/cypress/display.test.js

```diff
@@ -0,0 +1,26 @@
+beforeEach(() => {
+    cy.visit("localhost:3000");
+});
+
+describe('Find authors at smashing', () => {
+    it('Find the author Ramona Schwering', () => {
+        cy.log('This is our brand-new test');
+        cy.get('.headline-content', { timeout: 10000 }).should('be.visible');
+        cy.get('.main').contains('NEWS').then(($myElement) => {
+            //doSomething($myElement)
+            //cy.get($myElement)..should('be.visible').and($img)
+            //cy.get('div[class="image"]').find("img").should('be.visible');
+            cy.get('$myElement[class="image"]').find("img").should('be.visible');
+          });
+        cy.get('.main').contains('SPORT');
+        cy.get('.main').contains('POLITICS');
+        cy.get('.main').contains('WORLD');
+        cy.get('.main').contains('GARDENING');
+        cy.get('.main').contains('MOTORING');
+        cy.get('.main').contains('BUSINESS');
+        cy.get('.main').contains('CULTURE');
```

Comment:
Our categories come from the database. You might need to make a call to `localhost:3001/categories.list` to determine what categories should be displayed on the page.